apps/prairielearn/src/pages/instructorCourseAdminStaff/instructorCourseAdminStaff.html.ts

```diff
@@ -596,7 +596,12 @@ function StaffTable({
                                 </button>
                               </div>
                             </div>
-                            <button type="submit" class="btn btn-sm btn-outline-primary">
+                            <button
+                              type="submit"
+                              title="Remove access"
```

Comment:
Title is not necessary here. It's only used on the other buttons because they trigger popovers and Bootstrap can pull the popover's title from the `title` attribute. I get how this is confusing though; read my other comment suggesting we use `data-title` for that instead (though not in this PR).

Reply:
Removed title in [3b5665b](https://github.com/PrairieLearn/PrairieLearn/pull/10496/commits/3b5665b6bc0be4706a91dfd033bf10fb8b256e2e).